bazel_tools/sh/sh.bzl

```diff
@@ -38,9 +39,8 @@ _sh_inline_script = rule(
         "output": attr.output(
             mandatory = True,
         ),
-        "_template": attr.label(
```

Comment:
Maybe add a comment why this is not a private attribute?

Reply:
It is a private attribute again.